vulnerabilities/importers/fireeye.py

```diff
@@ -53,7 +53,7 @@ def advisory_data(self) -> Iterable[AdvisoryData]:
                 if Path(file).stem == "README":
                     continue
                 try:
-                    with open(file) as f:
+                    with open(file, encoding="utf-8-sig") as f:
```

Comment:
I was receiving this small error in `vulnerabilities.importers.fireeye.FireyeImporter` 
turns out this was caused by wrong encoding type. Since, this was not a big issue so I fixed it here in commit [145cb4b](https://github.com/nexB/vulnerablecode/pull/1378/commits/145cb4b510c7259fb70387ccf6778c0c30f5a480)

Error:-
```
Importing data using vulnerabilities.importers.fireeye.FireyeImporter
        Fetching Advisories █████∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙ 17%
Traceback (most recent call last):
  File "/home/harsh/vulnerablecode/vulnerabilities/management/commands/import.py", line 64, in import_data
    ImportRunner(importer).run()
  File "/home/harsh/vulnerablecode/vulnerabilities/import_runner.py", line 61, in run
    count = self.process_advisories(advisory_datas=advisory_datas, importer_name=importer_name)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/harsh/vulnerablecode/vulnerabilities/import_runner.py", line 95, in process_advisories
    for data in advisory_datas:
  File "/home/harsh/vulnerablecode/vulnerabilities/importers/fireeye.py", line 57, in advisory_data
    yield parse_advisory_data(raw_data=f.read(), file=file, base_path=base_path)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/harsh/vulnerablecode/vulnerabilities/importers/fireeye.py", line 79, in parse_advisory_data
    md_dict = md_list_to_dict(md_list)
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/harsh/vulnerablecode/vulnerabilities/importers/fireeye.py", line 153, in md_list_to_dict
    md_dict[md_key].append(md_line)
    ~~~~~~~^^^^^^^^
KeyError: ''
Failed to run importer vulnerabilities.importers.fireeye.FireyeImporter. Continuing...
Fetching Data from Databases |████████████████████████████████| 1/1
CommandError: 1 failed!: vulnerabilities.importers.fireeye.FireyeImporter

```

Please review @TG1999 and @ziadhany

Reply:
@harsh098 `MNDT-2023-0017.md , MNDT-2023-0019.md` has a problem with encoding I think .
I don't think this pull request should to handle this issue , just create a issue for this in [vulnerablecode/issues](https://github.com/nexB/vulnerablecode/issues) and drop this commit [145cb4b](https://github.com/nexB/vulnerablecode/pull/1378/commits/145cb4b510c7259fb70387ccf6778c0c30f5a480)

Reply:
ok @ziadhany

Reply:
To save the maintainers' time I have opened an issue and made a PR  
please have a look at issue #1403 and PR  #1404